shared/src/popup.html

```diff
@@ -83,6 +83,23 @@
         </span>
       </div>
 
+      <div id="privacy_consent_message" class="setting_row" style="display: none">
+        <div>
+          <div class="title" style="font-size: 1.2rem; margin-bottom: 1rem; text-align: center;">
+            Privacy Notice
+          </div>
+          <div class="desc" style="font-size: 0.9rem;">
+            If you allow it to, this extension will access your kagi.com session cookie and use it to automatically configure the extension and inject that session header in all requests to kagi.com, including in incognito mode (if you manually and explicitly allow it access).
+            <br /><br />
+            If you allow access to your currently active tab, you can use the Universal Summarizer and that will send your currently active tab's URL to kagi.com.
+            <br /><br />
+            <a href="https://help.kagi.com/kagi/faq/web-extension-privacy.html" target="_blank" rel="noopener noreferrer">Read the full privacy policy</a>
```

Comment:
@vprelovac this URL doesn't exist, and IANAL, but [they demand](https://extensionworkshop.com/documentation/publish/add-on-policies/#data-disclosure-collection-and-management) it's exclusive to the extension, so I can't link to Kagi's "normal" privacy policy.

If you need this to be in a separate URL, let me know and I'll update it here and rebuild the ZIPs.

Reply:
We have updated this URL with [https://kagi.com/privacy#browser-extension](https://kagi.com/privacy#browser-extension) in separate commit.